src/building/intro.md

```diff
@@ -1,4 +1,37 @@
 # How to build and run the compiler
 
+Please see [prerequisites](prerequisites.md) for setting up Rust and Node.js.
 
 ## zx —— The build tool Rspack using
+
+### Running setup script
+
+Make sure you are under the workspace root
+
+```bash
+node ./scripts/meta/setup.js
+```
+
+### Install Node.js dependencies
+
+Install Node.js dependencies via [pnpm](https://pnpm.io/).
+
+```bash
+# enable pnpm with corepack, only available on node >= `v14.19.0`
+corepack enable
```

Comment:
This should not be required since we already have the `setup` script. But we don't need to remove it in this PR.